mythril/support/truffle.py

```diff
@@ -145,7 +145,11 @@ def get_sigs_from_truffle(sigs, contract_data):
 
 def get_mappings(source, deployed_source_map):
     mappings = []
+    prev_item = ""
     for item in deployed_source_map:
```

Comment:
```suggestion
     for item in [e for e in deployed_source_map if e]:
```

Reply:
I think this allows you to not use the prev_item stuff here

Reply:
But in case of mappings we can't ignore when e=="", as that would miss a mapping right?(when e=="" then solidity says to use the previous map for that point too) We need to add them to the mappings list. This might ignore such mappings.